### DISCUSSION TITLE
Arrows for energy flow

### DISCUSSION THREAD

**Discussion by github-daniel-stoian** [slipx06/sunsynk-power-flow-card#435] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Energy flow is represented by moving dots on a flat line from sources to destination.
The direction of the moving dots is representing the energy flow.
But you need to look at is for a couple of seconds to understand what are the directions/flows.

<img width="1042" alt="Screenshot 2024-05-13 at 07 51 15" src="https://github.com/slipx06/sunsynk-power-flow-card/assets/26714611/7bd96d7d-8d12-4c6d-adcd-4a289b99e646">

### Expected behaviour

Option to represent energy flow by moving arrows on a flat line from sources to destination.
The direction of the moving arrows is representing the energy flow.

Or

Option to represent energy flow by "arrow snakes" (instead of flat lines) from sources to destination.
The direction of the moving "arrow snakes" is representing the energy flow.

You can tell from the first sight what are the directions.
<img width="1068" alt="Screenshot 2024-05-13 at 07 52 13" src="https://github.com/slipx06/sunsynk-power-flow-card/assets/26714611/7301f9bb-cf60-4e4e-8773-d75376cb5f04">

### Mode

lite

### Context / Reason

Better understand the energy flow quickly.

**Comment by slipx06** [comment]
Thanks for the suggestion but not something I can implement using svg marker element with the current animation logic. Will park this as a future idea
